### DISCUSSION TITLE
is it possible to specify different reader when using LoadImaged

### DISCUSSION THREAD

**Discussion by jxchen01** [Project-MONAI/MONAI#4492] [General]
when `LoadImaged` has several keys, is it possible to use different readers for different keys? for example, input image needs to be loaded by one reader, but ground truth is in different format and therefore needs to use a different loader?

**Comment by ebrahimebrahim** [comment]
In order to achieve this I usually compose two reader transforms consecutively, one targeting one key and one targeting the other key.

**Reply by jxchen01** [reply]
aha!! nice! thanks!
